packages/cli/src/util/deploy/process-deployment.ts

```diff
@@ -127,6 +123,24 @@ export default async function processDeployment({
           total: missingSize,
           clear: true,
         });
+
+        bar.tick(0);
+
+        uploads.forEach((e: any) =>
+          e.on('progress', () => {
+            if (!bar) return;
```

Comment:
**question (non-blocking):** When would `bar` be falsey? Is this here just to satisfy TS?

Reply:
Never, and ya.